demos/client-example/env.development

```diff
@@ -7,4 +7,5 @@ PROSOPO_PORT=9230
 PROSOPO_DEFAULT_NETWORK=development
 PROSOPO_DEFAULT_ENVIRONMENT=development
 PROSOPO_MONGO_EVENTS_URI=mongodb+srv://<MONGO_URI_HERE>/frictionless_events
-_DEV_ONLY_WATCH_EVENTS=false
\ No newline at end of file
+_DEV_ONLY_WATCH_EVENTS=false
+PROSOPO_HONEYPOT=true
```

Comment:
How will this work in github CICD workflow? The honeypot will always be enabled.